src/main/kotlin/org/rust/ide/refactoring/RsRenameProcessor.kt

```diff
@@ -32,16 +33,19 @@ class RsRenameProcessor : RenamePsiElementProcessor() {
 
     override fun renameElement(element: PsiElement, newName: String, usages: Array<out UsageInfo>, listener: RefactoringElementListener?) {
         if (element is RsPatBinding) {
-            usages
-                .filter {
-                    val usageElement = it.element
-                    usageElement is RsStructLiteralField && usageElement.colon == null
+            usages.forEach {
+                val usageElement = it.element
+                if (usageElement is RsStructLiteralField && usageElement.colon == null) {
+                    val newPatField = RsPsiFactory(element.project).createStructLiteralField(element.text, newName)
+                    usageElement.replace(newPatField)
+                    return@forEach
                 }
-                .forEach {
-                    val newPatField = RsPsiFactory(element.project)
-                        .createStructLiteralField(element.text, newName)
-                    it.element!!.replace(newPatField)
+                val field = PsiTreeUtil.getParentOfType(usageElement, RsStructLiteralField::class.java)
```

Comment:
Unfortunately, it doesn't work in the following case:
```rust
struct Foo {
    value: u32,
}

fn bar(/*caret*/ bar: u32) -> Foo {
    Foo {
        value: foo(bar),
    }
}

fn foo(value: u32) -> u32 { unimplemented!() }
```
If you rename `bar` to `value`, you change produces 

```rust
fn bar(value: u32) -> Foo {
    Foo { value }
}
```
instead of

```rust
fn bar(value: u32) -> Foo {
    Foo { value: foo(value) }
}
```

We need to check if only a particular ancestor is `RsStructLiteralField` instead of searching any ancestor of this type